test/ts/migrations/3_deploy_test_contracts_test.ts

```diff
@@ -31,6 +31,7 @@ contract("Migration #3: Deploying Test Contracts", async (ACCOUNTS) => {
     const REP_TOKEN_SYMBOL = "REP";
     const MKR_TOKEN_SYMBOL = "MKR";
     const ZRX_TOKEN_SYMBOL = "ZRX";
+    const WRAPPED_ETH_SYMBOL = "WETH";
```

Comment:
It'd be nice if all these values were accessible globally.

In Dharma.js, we specify in our constants, the following:

```export namespace ERC20TokenSymbol {
    export const REP = "REP";
    export const ZRX = "ZRX";
    export const MKR = "MKR";
}```

Reply:
Hmmm, yeah I agree though I don't know if this PR is the time to do this. I bet we use these few symbols all across testing files.

Reply:
Yeah, fair enough, just pointing it out. You have my approval already.